script/playbackstrategy/legacyplayeradapter.js

```diff
@@ -247,6 +247,7 @@ define('bigscreenplayer/playbackstrategy/legacyplayeradapter',
           mediaPlayer.initialiseMedia('video', mediaSources.currentSource(), mimeType, playbackElement, setSourceOpts);
           if (mediaPlayer.beginPlaybackFrom && !isPlaybackFromLivePoint) {
             currentTime = startTime;
+            DebugTool.keyValue({key: 'initial-playback-time', value: startTime + timeCorrection});
```

Comment:
Is it worth adding the || 0 here to keep the log consistent with the code.

Reply:
I want to see when the value is out of range or undefined.